lib/maxitest/global_must.rb

```diff
@@ -1,8 +1,7 @@
-# Allow global must_* assertion style without deprecations
+# Allow .must_* and .wont_* assertions on all objects
```

Comment:
unrelated , i wanted to add the `# frozen_string_literal: true` in all files, but i dunno about your style.

Reply:
oh yeah that's good too

Reply:
https://github.com/grosser/maxitest/pull/85